pkg/controller/shared/shared_informer.go

```diff
@@ -81,13 +87,26 @@ type sharedInformerFactory struct {
 	customListerWatchers  ListerWatcherOverrides
 	defaultResync         time.Duration
 
+	templateInformers      templateinformers.SharedInformerFactory
```

Comment:
I don't like this. If we must have a new struct, we should be building out a new struct that doesn't carry the legacy of hand-coded informers with it.